WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -591,8 +591,7 @@ public void run() {
 
     private void initializePostObject() {
         if (mPost != null) {
-            mOriginalPost = mPost.clone();
-            mOriginalPostHadLocalChangesOnOpen = mOriginalPost.isLocallyChanged();
+            mPostSnapshotWhenEditorOpened = mPost.clone();
```

Comment:
> This is a ticking bomb, as this means it will be subject to triggering a NPE at any of the places where `mPostSnapshotWhenEditorOpened`'s methods are being accessed.

Good point, note that this was the same previously but because `mOriginalPost` was updated quite often (onSave) it was harder to trigger such NPEs. I'll null check calls to `.isLocallyChanged()`.

I'll also check `isFirstTimePublish()` calls.

Reply:
Updated in 0d510aa48e972209bf7608876c0fd2d59083c238 I added a `@Nullable` annotation and ran the associated code inspection to check for other of these issues.